src/main/java/io/sprucehill/zalando/api/nativecart/model/Order.java

```diff
@@ -0,0 +1,98 @@
+package io.sprucehill.zalando.api.nativecart.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.sprucehill.zalando.api.model.Base;
+
+/**
+ * 
+ * @author dipteewarudkar
+ *
+ */
+public class Order extends Base {
+
+	@JsonProperty(value="order_number")
+	private String orderNumber;
+
+	@JsonProperty(value="customer_number")
+	private String customerNumber;
+
+	@JsonProperty(value="billing_address")
+	private Address billingAddress	;
+
+	@JsonProperty(value="shipping_address")
+	private Address shippingAddress;
+
+	@JsonProperty
+	private Delivery delivery;
+
+	@JsonProperty(value="gross_total")
+	private Money grossTotal;
+
+	@JsonProperty(value="tax_total")
+	private Money taxTotal;
+
+	@JsonProperty
+	private String created;
```

Comment:
We should maybe use `Date` here instead of `String` and apply correct `@JsonFormat`